client/packages/requisitions/src/ResponseRequisition/DetailView/Toolbar.tsx

```diff
@@ -65,6 +68,11 @@ export const Toolbar: FC = () => {
                   />
                 }
               />
+              { !linkedToShipment && (
```

Comment:
can you also check the disabled state here - currently the message shows when finalised, and it isn't necessary then 😉 

<img width="588" alt="image" src="https://user-images.githubusercontent.com/9192912/218632489-8d35f1b6-188d-4381-b753-b53c7382776d.png">